riseagain47/EPIC-Mail
# Ft user create group #164725374

#### What does this PR do?
- This PR adds a create group endpoint to the project

#### Description of Task to be completed?
Have the following:
- group model
- group controlller 
- group route
- group unit and integration tests

#### How should this be manually tested?
1. User should start the server with `npm run dev-start`
2. Open Postman and make a POST HTTP request to `localhost:5000/api/v2/groups
3. User should receive a success message upon success and validation error message if fields are not completed

#### Any background context you want to provide?
User should that the server is running before making the POST HTTP request

#### What are the relevant pivotal tracker stories?
- #164725374 - Feature - User  can create and own a group
